src/Microsoft.Extensions.FileSystemGlobbing/Abstractions/DirectoryInfoWrapper.cs

```diff
@@ -7,17 +7,31 @@
 
 namespace Microsoft.Extensions.FileSystemGlobbing.Abstractions
 {
+    /// <summary>
+    /// Wraps an instance of <see cref="System.IO.DirectoryInfo" /> and provides implementation of
+    /// <see cref="DirectoryInfoBase" />
+    /// </summary>
     public class DirectoryInfoWrapper : DirectoryInfoBase
     {
         private readonly DirectoryInfo _directoryInfo;
         private readonly bool _isParentPath;
 
+        /// <summary>
+        /// Initializes an instance of <see cref="DirectoryInfoWrapper" />
+        /// </summary>
+        /// <param name="directoryInfo">The <see cref="System.IO.DirectoryInfo" /></param>
+        /// <param name="isParentPath">
+        /// <c>true</c> when the <paramref name="directoryInfo" /> should be represented as the parent
+        /// directory with '..'
+        /// </param>
+        // TODO should isParentPath be exposed publically? This seems like an internal implementation detail
```

Comment:
Actually can the entire class become internal if all it do is to implement `DirectoryInfoBase`?

_Note: not PR blocking, we can do it later._